core/templates/dev/head/services/explorationServices.js

```diff
@@ -19,7 +19,7 @@
 
 // A service that provides a number of utility functions useful to both the
 // editor and player.
-oppia.factory('oppiaExplorationHtmlFormatterService', [
```

Comment:
We should rename this file too, right (and change the fileoverview) since there's just one service in it?

Also, I'd suggest getting rid of the "oppia" prefix. We can just call this ExplorationHtmlFormatterService. (Sorry for not detecting this at the time of writing the issue.)

Reply:
@seanlip I will surely look into this matter and make the necessary updates as soon as travis finishes the checks.